examples/golang/components/sqldb-postgres-query/postgres.go

```diff
@@ -33,30 +37,36 @@ func init() {
 }
 
 func (c Component) Call() string {
-	query := interfaces.WasmcloudPostgres0_1_0_draft_QueryQuery(CREATE_TABLE_QUERY, make([]PgValue, 0))
+	query := Query(CREATE_TABLE_QUERY, make([]PgValue, 0))
 	if query.IsErr() {
 		return fmt.Sprintf("ERROR: failed to create table: %v", query.UnwrapErr())
 	}
+
 	val := interfaces.WasmcloudPostgres0_1_0_draft_TypesPgValueText("inserted example go row!")
-	insertResult := interfaces.WasmcloudPostgres0_1_0_draft_QueryQuery(INSERT_QUERY, []PgValue{val})
+	insertResult := Query(INSERT_QUERY, []PgValue{val})
 	if insertResult.IsErr() {
 		return fmt.Sprintf("ERROR: failed to insert row: %v", insertResult.UnwrapErr())
 	}
-	insertedRows := insertResult.Unwrap()
-	var rowEntry []RowEntry
-	if len(insertedRows) == 1 {
-		rowEntry = insertedRows[0]
-	} else {
-		return "ERROR: failed to insert row"
-	}
 
-	for _, row := range rowEntry {
-		if row.ColumnName == "description" {
-			return fmt.Sprintf("SUCCESS: inserted and retrieved: %v", row.Value.GetText())
-		}
+	selectResult := Query(SELECT_QUERY, make([]PgValue, 0))
+	if selectResult.IsErr() {
+		return fmt.Sprintf("ERROR: failed to select rows: %v", selectResult.UnwrapErr())
+	}
+	selectedRows := selectResult.Unwrap()
+	if len(selectedRows) > 0 {
+		firstRow := selectedRows[0]
+		return fmt.Sprintf("SUCCESS: we selected a row! %v", firstRow)
```

Comment:
So by default you'll get the same row every time here -- maybe you want to take the last one (since this is pulling all the rows anyway)? You could also apply a `LIMIT` and name this `SELECT_FIRST_WITH_DESCRIPTION` or something